python/pyiceberg/cli/output.py

```diff
@@ -127,6 +128,33 @@ def describe_table(self, table: Table) -> None:
         output_table.add_row("Properties", table_properties)
         Console().print(output_table)
 
+    def describe_refs(self, ref_tups: List[Tuple[str, SnapshotRef]]):
+        refs_table = RichTable(title="Snapshot Refs")
+        refs_table.add_column("Type")
+        refs_table.add_column("Ref")
+        refs_table.add_column("Max ref age ms")
+        refs_table.add_column("Min snapshots to keep")
+        refs_table.add_column("Max snapshot age ms")
+
+        for ref_tup in ref_tups:
+            name = ref_tup[0]
+            ref = ref_tup[1]
+            if ref.snapshot_ref_type == 'branch':
+                min_snapshots_to_keep = str(ref.min_snapshots_to_keep)
```

Comment:
Still thinking through what's the best way to surface to a user these properties aren't set. I wonder if the current outputting of "None" will confuse users

Reply:
I agree that it is best to pull in the table properties, and show the fall back expiration time.